libs/systemctl/systemd/template.service.in

```diff
@@ -1,10 +1,10 @@
 [Unit]
```

Comment:
![image](https://github.com/Skaginn3x/framework/assets/15651464/6507a86d-4261-4add-b8ee-76020462a3db)


https://fedoramagazine.org/systemd-template-unit-files/

Reply:
`sudo systemctl start ipc-ruler.service`
`sudo systemctl start ethercat.service`
`sudo systemctl start sensor_control@id.service`

Reply:
rewritten so that service files are called

`tfc@ethercat.service`
`tfc@ipc-ruler.service`
etc...

Reply:
How can we edit id and or log level?

Reply:
have you tested if this works like it is now?

Reply:
> How can we edit id and or log level?

one service file is created for each respective executable

Reply:
> How can we edit id and or log level?

do you want to set the log level while the service file is running or when it is created?

Reply:
> have you tested if this works like it is now?

I have ran the cmake command to create the .deb package, the service files get created and they run correctly when I try them on the raspberry pi

Reply:
https://www.freedesktop.org/software/systemd/man/systemd.service.html, under "Service Templates"

systemd service files can take a single argument as template arguments but not more than one. I think a good solution is to create a template file for each log-level where the id is the only template parameter. That is the only variable that is infinitely redefineable.